diff --git a/net/minecraft/server/players/PlayerList.java b/net/minecraft/server/players/PlayerList.java
--- a/net/minecraft/server/players/PlayerList.java
+++ b/net/minecraft/server/players/PlayerList.java
@@ -161,6 +161,7 @@
             player.getY(),
             player.getZ()
         );
+        LOGGER.info("{} joined the game!", player.getGameProfile().name());
         LevelData levelData = level.getLevelData();
         ServerGamePacketListenerImpl playerConnection = new ServerGamePacketListenerImpl(this.server, connection, player, cookie);
         connection.setupInboundProtocol(
@@ -309,6 +310,7 @@
     public void remove(final ServerPlayer player) {
         ServerLevel level = player.level();
         player.awardStat(Stats.LEAVE_GAME);
+        LOGGER.info("{} left the game.", player.getGameProfile().name());
         this.save(player);
         if (player.isPassenger()) {
             Entity vehicle = player.getRootVehicle();
