diff --git a/net/minecraft/server/dedicated/DedicatedServer.java b/net/minecraft/server/dedicated/DedicatedServer.java
index 62adce2..c1ea72b 100644
--- a/net/minecraft/server/dedicated/DedicatedServer.java
+++ b/net/minecraft/server/dedicated/DedicatedServer.java
@@ -432,6 +432,7 @@ public class DedicatedServer extends MinecraftServer implements ServerInterface
     public void handleConsoleInputs() {
         while (!this.consoleInput.isEmpty()) {
             ConsoleInput input = this.consoleInput.remove(0);
+            LOGGER.info("CONSOLE issued server command: {}", input.msg);
             this.getCommands().performPrefixedCommand(input.source, input.msg);
         }
     }
