diff --git a/net/minecraft/server/MinecraftServer.java b/net/minecraft/server/MinecraftServer.java
index 29952d3..c679e30 100644
--- a/net/minecraft/server/MinecraftServer.java
+++ b/net/minecraft/server/MinecraftServer.java
@@ -198,7 +198,8 @@ import org.slf4j.Logger;
 
 public abstract class MinecraftServer extends ReentrantBlockableEventLoop<TickTask> implements CommandSource, ServerInfo, ChunkIOErrorReporter {
     private static final Logger LOGGER = LogUtils.getLogger();
-    public static final String VANILLA_BRAND = "vanilla";
+    public static final String VANILLA_BRAND = "VeltisMC";
+    public static volatile MinecraftServer SERVER;
     private static final float AVERAGE_TICK_TIME_SMOOTHING = 0.8F;
     private static final int TICK_STATS_SPAN = 100;
     private static final long OVERLOADED_THRESHOLD_NANOS = 20L * TimeUtil.NANOSECONDS_PER_SECOND / 20L;
@@ -371,6 +372,7 @@ public abstract class MinecraftServer extends ReentrantBlockableEventLoop<TickTa
         this.clockManager.init(this);
         this.customBossEvents = this.savedDataStorage.<CustomBossEvents>computeIfAbsent(CustomBossEvents.TYPE);
         this.scheduledEvents = this.savedDataStorage.<TimerQueue<MinecraftServer>>computeIfAbsent(TimerQueue.TYPE);
+        SERVER = this;
     }
 
     protected abstract boolean initServer() throws IOException;
@@ -1236,7 +1238,7 @@ public abstract class MinecraftServer extends ReentrantBlockableEventLoop<TickTa
     }
 
     public String getServerModName() {
-        return "vanilla";
+        return "VeltisMC (" + SharedConstants.getCurrentVersion().name() + ")";
     }
 
     public ServerClockManager clockManager() {
